m4_sessions/1801-inning/group_3.md

```diff
@@ -5,15 +5,29 @@ Group Members: Maria St. Louis-Sanchez, Katie Scruggs, Julie Hawkins, Jason Hugh
 
 Name of Session: Relaxation in Practice
 
+[Link to our Slideshow](https://docs.google.com/presentation/d/1_upUNqI9hpMqXY5ljfxlmVtSN1eB2Vkjlh9Qgn-m0xU/edit?usp=sharing)
+
 Session Description (Why should students come to this session):  Students who come to this session will learn and practice techniques to help them relax during stressful times. They will also reflect on what strategies will work for them (or already work for them), and how to recognize when it is time to use these techniques and step away from the computer.
 
 Session Outcomes (What will students learn/leave with?): Students will leave with an awareness of how important it is to make time to do activities that relax you. They will also have a clear list of activities they're interested in, and a Slack channel set up to share ideas and for support.
 
 Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
 
-Intro (10 min)- Lights will be off and calming music will be playing. Welcome to the session, facilitators will introduce ourselves. Icebreaker = what do you already do to relax? what makes you feel like you?
+Intro (10 min) [Katie]- Lights will be off and calming music will be playing when people walk in. Welcome to the session, facilitators will introduce ourselves. Icebreaker = what do you already do to relax? what makes you feel like you?
+
+Mindfulness Video (10 min) - [https://www.youtube.com/watch?v=qzR62JJCMBQ](https://www.youtube.com/watch?v=qzR62JJCMBQ)
+
+What is Stress / Ideas for Self-Care (10 min) [Julie] We will have students turn and talk to each other about their answers to the question: What is stress? We will then share categories of self-care that can help mitigate the results of stress.
 
-Meditation Session (5)- We will have a small mindfulness meditation session via YouTube Video.
+Stations for Self-Care [introduced by Maria] (7 min each)
+Each of us will be at a station, where students will brainstorm self-care activities they are interested in implmenting.and y
+  * mental
+  * physical
+  * practial
+  * spiritual/emotional
+  
+Sharing results of stations (5 min) [Maria]
```

Comment:
I'd love to know what PREVENTS students from being able to do these things.  I feel like they know what they should do, but still don't do it.